newIDE/app/src/AssetStore/AssetDetails.js

```diff
@@ -204,6 +213,35 @@ export const AssetDetails = React.forwardRef<Props, AssetDetailsInterface>(
       [loadAsset]
     );
 
+    const loadAuthorPublicProfiles = React.useCallback(
+      async () => {
+        try {
+          const authorIds: Array<string> = (asset && asset.authorIds) || [];
+          if (authorIds.length === 0) return;
+          console.log('fetching', asset);
+          const userPublicProfileByIds = await getUserPublicProfilesByIds(
+            authorIds
+          );
+          console.log(userPublicProfileByIds);
+          const userPublicProfiles = Object.keys(userPublicProfileByIds).map(
```

Comment:
Maybe easier to have `Object.values`?

Reply:
I wish, but Object.values is painful and loses the type of the element

Reply:
And it seems more performant as well: https://www.zhenghao.io/posts/object-vs-map